internal/xds/translator/translator.go

```diff
@@ -253,8 +258,8 @@ func processUDPListenerXdsTranslation(tCtx *types.ResourceVersionTable, udpListe
 
 }
 
-// findXdsListener finds a xds listener with the same address, port and protocol, and returns nil if there is no match.
-func findXdsListener(tCtx *types.ResourceVersionTable, address string, port uint32,
+// findXdsListenerByAddress finds a xds listener with the same address, port and protocol, and returns nil if there is no match.
+func findXdsListenerByAddress(tCtx *types.ResourceVersionTable, address string, port uint32,
```

Comment:
```suggestion
func findXdsListenerByAddress(tCtx *types.ResourceVersionTable, host string, port uint32,
```
address = schema+host+port

Reply:
how about naming  it to `findXdsListenerByHostPort` ?

Reply:
I'm ok with the name, cause most of time you don't care scheme.